ajax/widget/js/widget.js

```diff
@@ -0,0 +1,41 @@
+let employeeList = new XMLHttpRequest();
+employeeList.onreadystatechange = function () {
```

Comment:
This is an anonymous function.
I think you mostly want to use arrow functions nowadays.

Reply:
I actually Just learned this yesterday! Done.